libs/localization/assets/locales/en/sheet.json

```diff
@@ -149,5 +151,16 @@
     "skill": "Skill DMG Mult."
   },
   "energy": "Energy",
-  "triggerQuota": "Trigger Quota"
+  "triggerQuota": "Trigger Quota",
+  "enemyAffected": {
```

Comment:
Not action item, but perhaps situations like this can be simplified using [nesting](https://www.i18next.com/translation-function/nesting)?

Reply:
Probably worth doing here, these strings feel like a perfect fit. I'll see how hard it is and revise the pr

Reply:
I added it. We could (or should?) migrate/consolidate all translations like this moving forward with interpolation maybe, instead of having a translator translate a string multiple times (like for hitOp). Although that means strings might become more awkward to accommodate such things.

One English example I can think of is if we had a generic string "After triggering a {element} reaction". We can easily interpolate "hydro", "cryo", etc in there and be fine. But to interpolate "electro" in there, we would need to modify the base string to be "After triggering a**(n)** {element} reaction", which is a bit clunky. I don't know enough about other languages to know if this is something that could occur in them as well, probably with masculine/feminine nouns+verbs etc.

But probably still worth consolidating to save translation effort? In the future though

Reply:
I think you can add in `a` vs `an` handling using [formatting](https://www.i18next.com/translation-function/formatting)